app/views/publishers/vacancies/job_applications/self_disclosure/show.html.slim

```diff
@@ -0,0 +1,21 @@
+- content_for :page_title_prefix, t(".page_title")
+
+- content_for :breadcrumbs do
+  = govuk_back_link text: t("app.back_link"), href: pre_interview_checks_organisation_job_job_application_path(vacancy.id, job_application), html_attributes: { "aria-label" => "Back navigation", role: "navigation" }
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    span.govuk-caption-l = @self_disclosure.header_text
+    h1.govuk-heading-xl class="govuk-!-margin-bottom-5 govuk-!-margin-top-0"
+      = @self_disclosure.applicant_name
+
+    - if @self_disclosure.request.completed?
+      = render "jobseekers/job_applications/self_disclosure/review/completed"
```

Comment:
I have been trying to always pass variables to partials rather than them accessing globals directly. I always thought it was a standard, but don't know where from?